ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -1034,6 +1035,22 @@ def test_source_ipahealthcheck_ds_backends(self, rename_ldif):
             assert check["result"] == "CRITICAL"
             assert exception_msg in check["kw"]["exception"]
 
+    def test_source_pki_server_clones_connectivity_and_data(self):
+        """
+        This testcase checks that when ClonesConnectivyAndDataCheck
+        is run it doesn't display source not found error
+        """
+        error_msg = (
+            "Source pki.server.healthcheck.clones.connectivity_and_data
+            not found"
+        )
+        result = self.master.run_command(
+            ["ipa-healthcheck", "--source",
+             "pki.server.healthcheck.clones.connectivity_and_data"]
+        )
+        for check in pki_clone_checks:
+            assert error_msg not in result.stdout_text
```

Comment:
The loop is not needed (error_msg is not modified by the loop), you can replace with:
```
         assert error_msg not in result.stdout_text 
```
Maybe you meant to ensure that all the checks in pki_clone_checks are displayed?
```
        for check in pki_clone_checks:
            assert check in result.stdout_text
```